css/at-rules/container.json

```
@@ -36,6 +36,102 @@
             "deprecated": false
           }
         },
+        "anchor_position_queries": {
+          "__compat": {
+            "description": "Anchor position queries (`anchored()`)",
+            "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#anchored-container-queries",
+            "support": {
+              "chrome": {
+                "version_added": "143"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "fallback": {
+            "__compat": {
+              "description": "`fallback` container feature",
```

Given that we're in the context of the `@container` at feature, mentioning "container" seems repetitive:

```suggestion
              "description": "`fallback` feature",
```